test/parallel/test-v8-coverage.js

```diff
@@ -35,6 +35,24 @@ function nextdir() {
   assert.strictEqual(fixtureCoverage.functions[0].ranges[1].count, 0);
 }
 
+// Outputs coverage when process.exit(1) exits process.
```

Comment:
perhaps, change to:

`// Outputs coverage when error is thrown in first tick`.

Reply:
Uh thanks for catching my copy-past error :)